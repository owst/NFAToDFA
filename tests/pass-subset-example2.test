0
0--a->0,1
1--b->1,2
2
