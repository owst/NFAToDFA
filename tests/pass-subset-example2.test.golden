digraph NFA {
rankdir = LR;
node [shape = point]; init
0 [label="{0}" shape="circle" ]; init -> 0;
1 [label="{0,1}" shape="circle" ];
0 -> 1 [label="a"];
2 [label="{1,2}" shape="doublecircle" ];
1 -> 2 [label="b"];
2 -> 2 [label="b"];
1 -> 1 [label="a"];
}
